.github/workflows/setup/action.yaml

```diff
@@ -5,6 +5,10 @@ inputs:
     description: 'Whether or not to pull the git lfs'
     required: false
     default: 'true'
+  cache_key_prefix:
+    description: 'Prefix for caching key'
+    required: false
+    default: 'scons'
   is_retried:
```

Comment:
why doesn't this use the new build action?

Reply:
im not sure if i follow, this only builds base images and restores caches, while build actually compiles stuff